autoload/lsp/omni.vim

```diff
@@ -303,54 +303,64 @@ function! lsp#omni#get_vim_completion_items(options) abort
     let l:start_characters = [] " The mapping of item specific start_character.
     let l:vim_complete_items = []
     for l:completion_item in l:items
-        let l:expandable = get(l:completion_item, 'insertTextFormat', 1) == 2
```

Comment:
Move `l:expandable` to location used here.

https://github.com/prabirshrestha/vim-lsp/blob/c0482f0fb407c29a88875616175784a886f131c3/autoload/lsp/omni.vim#L339